src/parser.ts

```diff
@@ -129,7 +139,7 @@ export function visit(node: unknown, visitor: ASTVisitor, nodeParent?: ASTNode):
 
   const selector = (node.type + ':exit') as `${ASTNodeTypeString}:exit`
   if (visitor[selector] !== undefined) {
-      // TODO can we avoid this `as any`
-    visitor[selector]!(node as any, nodeParent)
+    // TODO can we avoid this `as never`
+    visitor[selector]!(node as never, nodeParent)
```

Comment:
`typescript` now says that this value defaults to `never` I kept the comment as it's probably a very specific typing error that  changed as `typescript` got smarter with new versions